media/windows/ricoh-printer/ricoh_db_create.py

```diff
@@ -0,0 +1,121 @@
+#!/user/bin/env python3
+
+# Script which runs daily to update staff print totals
+# based on CSVs generated by the printer itself.
+
+import sys
+sys.path.insert(1, '../../python/')
+
+import os
+import re
+import csv
+import datetime
+import sqlite3
+from sqlite3 import Error
+
+#import ECC
+#import PDSChurch
+
+from pprint import pprint
+from pprint import pformat
+
+###########################################################
+
+ricoh_db = "media/windows/ricoh-printer/ricoh.db"
+today = datetime.date.today()
+datestring = f'{today.year}{today.month}{today.day}'
+debug_datestring = '20201013'
+
+def _strip(value):
+    #The CSV puts brackets around values, which we don't want
+    return value.strip('[').strip(']')    
+
+def load_latest_csv():
+    
+    filename = f'RICOH IM C4500_usercounter_{datestring}.csv'
+    
+    debug_filename = 'media/windows/ricoh-printer/RICOH IM C4500_usercounter_20201013.csv'
+    
+    csv_rows = list()
+
+    with open(debug_filename, encoding='utf-8') as csvfile: #TODO: change to filename when retrieval is automated
+        csvreader = csv.DictReader(csvfile)
+        for row in csvreader:
+            csv_rows.append(row)
+
+    print(f"== Loaded {len(csv_rows)} staffers from latest CSV")
+    return csv_rows
+
+def extract_csv_data(csv_rows):
+
+    def _extract_staffer(row, num, name):
+        return {
+            'num'           : num,
+            'name'          : name,
+            'total'         : int(row['Total Prints'].strip()),
+            'b&wtotal'      : int(row['B & W(Total Prints)'].strip()),
+            'colortotal'    : int(row['Color(Total Prints)'].strip()),
+        }
+
+    csv_staffers = list()
+    
+    for row in csv_rows:
+        try:
+            num = int(_strip(row['User']))
+            name = _strip(row['Name'])
+        except ValueError as e:
+            print(e)
+            num = 000
```

Comment:
I think you need `num` to be a string, not an integer.  The department numbers that you extract from the spreadsheet are going to be zero-prefixed.  If you store `010` as an integer, it will have the value of `10`, not `010`.

Reply:
A better word for `num` might be `department`.